build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'com.github.johnrengelman.shadow' version '5.2.0'
+}
+
+repositories {
+    maven {
+        url = 'http://artifacts.openmicroscopy.org/artifactory/maven'
+    }
+    maven {
+        url = 'http://repo.maven.apache.org/maven2'
+    }
+}
+
+dependencies {
+    implementation 'org.openmicroscopy:omero-server:5.5.6'
+    implementation 'com.google.guava:guava:27.1-jre'
```

Comment:
is that needed?

Reply:
also in other repo we have used implementation()

Reply:
Seems likely, can revisit once more code's written. Or just remove in a separate commit then I can revert when I use it.

Reply:
I've left this in since it was in the pom.xml that this was created from. We can update it in a later PR if necessary.